src/citrine/resources/team.py

```diff
@@ -13,7 +13,9 @@
 WRITE = "WRITE"
 READ = "READ"
 SHARE = "SHARE"
-TEAM_ACTIONS = Union[WRITE, READ, SHARE]
+CREATE = "CREATE"
+PUBLISH = "PUBLISH"
+TEAM_ACTIONS = Union[WRITE, READ, SHARE, CREATE, PUBLISH]
```

Comment:
@zoesyang Can you confirm that these team actions should be added?  I've seen them mentioned elsewhere so I'm guessing they're missing from this file but would like to double check.